src/components/Header/Header.jsx

```diff
@@ -109,24 +109,30 @@ const LinkGroup = ({ navbarPages }) => (
 );
 
 LinkGroup.propTypes = {
-  navbarPages: PropTypes.shape([{}]),
+  navbarPages: PropTypes.arrayOf(
+    PropTypes.shape({
+      key: PropTypes.string,
+      to: PropTypes.string,
+      label: PropTypes.string,
+    }),
+  ),
 };
 
 const SettingsGroup = ({ children }) => {
   const [anchorEl, setAnchorEl] = useState(null);
   const handleClose = useCallback(() => {
-    setAnchorEl(undefined);
+    setAnchorEl(null);
   }, [setAnchorEl]);
 
   return (
-    <>
+    <div>
```

Comment:
what's the purpose of this change?

Reply:
It's related to the ref forwarding in MUI. It doesn't work with fragments but should be with divs. It's still drops warning so... just a try.